concrete/src/Routing/Route.php

```diff
@@ -114,6 +114,14 @@ public function getMiddlewares()
         return $this->middlewares;
     }
 
-
+    /**
+     * Explicitly sets an OAuth2 scope to a route. This will be used if the route is consumed in an
+     * OAuth2 request.
+     * @param $scope
```

Comment:
:+1: on the comment, I'd just add what type `$scope` should be because there is no way to know at the moment